mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/chargedialog/ChargeDialogFragment.java

```diff
@@ -121,21 +121,22 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
 
         inflatedueDate();
         inflateChargesSpinner();
+        inflateChargeNameSpinner();
 
-        duedateString = charge_due_date.getText().toString();
+        duedateString = chargeDueDate.getText().toString();
         duedateString = DateHelper.getDateAsStringUsedForCollectionSheetPayload(duedateString)
                 .replace("-", " ");
 
-        bt_save_charge.setOnClickListener(new View.OnClickListener() {
+        btSaveCharge.setOnClickListener(new View.OnClickListener() {
```

Comment:
We are using ButterKhife, Remove the OnClickListener implementation and use ButterKhife @OnClick annotation